app/models/barcode/format_handlers.rb

```diff
@@ -126,7 +126,7 @@ def code128_barcode?
     end
 
     def code39_barcode?
-      %r{\A[A-Z0-9 \-\.$\/\+%]+\z}.match?(@human_barcode)
+      %r{\A[A-Z0-9 \-.$/+%]+\z}.match?(@human_barcode)
```

Comment:
@JamesGlover what are the removal of the backslashes about in these regexes? Has one of the updates caused a change in ruby regex syntax?